scripts/gendocs/postprocess/api-doc-postprocess.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+
+import { readFileSync, writeFileSync } from 'fs'
+import prettier from 'prettier'
+import matter from 'gray-matter'
+
+const apiDocsFile = readFileSync(0, 'utf8')
+
+const { content, data } = matter(apiDocsFile)
+let result = content
+result = prettier.format(`<div>${result}</div>`, {
+  parser: 'babel',
+  htmlWhitespaceSensitivity: 'strict',
+  proseWrap: 'never',
+  quoteProps: 'consistent',
+  bracketSameLine: true,
+  printWidth: 1000
+})
+
+result = result.split('\n').slice(1, -2).join('\n')
+result = result.replace(new RegExp(/{" "}/g), ' ')
```

Comment:
note: these bits are the magic for the refreshed website, written by Patrick. It makes sure the docs are parse-able by the MDX stuff.

Reply:
Correct. Without explicit code blocks, the parser throws an exception, as it awaits an expression with `{}`:

```
Could not parse expression with acorn: Unexpected content after expression

  136 | > 
  137 | >     $ kubectl create secret generic eab-secret --from-literal \
> 138 | >       secret={base64 encoded secret key}
      |                       ^
  139 |
  140 | An example of an ACME issuer with an External Account Binding is as follows.
```

By setting explicit code blocks/fences (```), everything works fine.